cassandra/delete_data.py

```diff
@@ -11,12 +11,17 @@
 import os
 import sys
 
-# Add parent directory to python path to locate the metadata_service_api package
-sys.path.insert(0, '/'.join((os.path.dirname(os.path.realpath(__file__)), '..')))
+# Add parent directory to python path to locate the
+# metadata_service_api package
+sys.path.insert(0, '/'.join(
+        (os.path.dirname(os.path.realpath(__file__)), '..')))
```

Comment:
Was checking style via pep8 checking tool (e.g. pep8 ~/ooi-tools/cassandra/delete_data.py) and that tool was not catching these indentation issues. Running with PyCharm now and correcting the PEP8 errors it calls out. Also reverting formatting changes caused by line length limitation discrepancy (80 vs. 120).